bedevere/bpo.py

```diff
@@ -104,15 +103,32 @@
             await gh.patch(event.data[event_name][body_location], data=body_data)
 
 
-def create_success_status(found_issue):
+def create_success_status(issue_number):
     """Create a success status for when an issue number was found in the title."""
-    issue_number = found_issue.group("issue")
     url = f"https://bugs.python.org/issue{issue_number}"
     return util.create_status(STATUS_CONTEXT, util.StatusState.SUCCESS,
                               description=f"Issue number {issue_number} found",
                               target_url=url)
 
 
+def create_failure_status_issue_not_on_bpo(issue_number):
+    """Create a failure status for when an issue does not exist on the bug tracker."""
+    description = f"Issue #{issue_number} not found on bugs.python.org"
+    url = "https://devguide.python.org/pullrequest/#submitting"
```

Comment:
What do you think of changing this to be the issue URL to show that it's a 404? If they specified an issue number they know that's what they should be doing so I don't think linking back to the docs is going to be the most helpful option for us.

Reply:
I totally agree. I will change the url to link to the (non-existing) issue.